pkg/api/common.go

```diff
@@ -127,6 +129,16 @@ func Validate(cfg *Config) error {
 	return cfg.Auth.Validate()
 }
 
+func getTenant(r *http.Request) (tenant string) {
+	if tn := r.Header.Get("TENANT"); tn == "" {
```

Comment:
for documentation there is no X prefix because of https://datatracker.ietf.org/doc/html/rfc6648

Reply:
Wrote as a comment.